### PR TITLE
controllers/admin i18n

### PR DESCRIPTION
**Pull request by yheuhtozr**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Provides i18n for app/controllers/admin files.

## Related Tickets & Documents

- Related Issue #14888
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [X] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by msarit** [comment]
@yheuhtozr Hello! Please resolve conflicts on this PR thanks!

File: app/controllers/admin/articles_controller.rb [hunk 1]
```diff
@@ -55,7 +55,7 @@ def unpin
 
       respond_to do |format|
         format.html do
-          flash[:success] = "Article Pinned!"
+          flash[:success] = I18n.t("admin.articles_controller.pinned")
```
Comment (aitchiss) [inline app/controllers/admin/articles_controller.rb:58]: This isn't a new issue, so definitely not blocking this PR, but there seems to be a logic flaw here - this action is _unpinning_ a post, but we display "Article Pinned!" as if the admin had just pinned it 🤔 

I don't mind if we fix this in a later PR

**Review by aitchiss — APPROVED** [review]
Thank you for this! 🔥

File: app/controllers/admin/articles_controller.rb (hunk 1 above)
Comment (yheuhtozr) [reply app/controllers/admin/articles_controller.rb:58]: Hi, I'm not familiar with your internal roadmap so I'll be happy to follow your instructions on whether it should be taken care within this PR.

Comment (aitchiss) [reply app/controllers/admin/articles_controller.rb:58]: Let's leave it for a later PR - I don't want it to hold up this one! I'll capture it in an issue 🙂

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -216,17 +221,19 @@ def send_email
...
-          render json: { error: "Both subject and body are required!" },
+          render json: { error: I18n.t("admin.users_controller.parameter_missing") },
                  content_type: "application/json",
                  status: :unprocessable_entity
         end
       end
     end
 
+    # NOTE: [@rhymes] This should be eventually moved in Admin::Users::Tools::EmailsController
+    # once the HTML response isn't required anymore
```
Comment (juliannatetreault) [inline app/controllers/admin/users_controller.rb:232]: **Quick suggestion:** We no longer need this comment, as `Admin::Users::Tools::EmailsController` no longer exists - let's remove it!:
```suggestion
```

File: config/locales/controllers/admin/en.yml [hunk 3]
```diff
@@ -1,18 +1,153 @@
...
+      privacy_policy:
+        title: Privacy Policy
+        description: A page that describes the privacy policy
+      terms_of_use:
+        title: Terms of Use
+        description: A page that describes the terms of use for the application
+    podcasts_controller:
+      new_owner_added: New owner added!
+      no_such_user: No such user
+      scheduled: 'Podcast''s episodes fetching was scheduled (%{title}, #%{id})'
```
Comment (juliannatetreault) [inline config/locales/controllers/admin/en.yml:82]: Is there a reason for the double single quotes here and [here](https://github.com/forem/forem/pull/17085/files#diff-3e3bb58d0be9d890324bbc64f8710e7ebf7acd5e1ecf759bb2e486851269b809R82) on line 82 of `fr.yml`? Could we wrap this message in double quotes and use one single quote instead?
```suggestion
      scheduled: "Podcast's episodes fetching was scheduled (%{title}, #%{id})"
```

File: config/locales/controllers/admin/fr.yml [hunk 4]
```diff
@@ -1,18 +1,153 @@
...
+    listing_categories_controller:
+      created: Listing Category has been created!
+      updated: Listing Category has been updated!
+      deleted: Listing Category has been deleted!
+    listings_controller:
+      updated: Listing updated successfully
+      destroyed: "'%{title}' was destroyed successfully"
+      no_credit: Not enough available credits
+    mods_controller:
+      trusted: "%{username} now has Trusted role!"
```
Comment (juliannatetreault) [inline config/locales/controllers/admin/fr.yml:53]: **Quick, non-blocking suggestion (same as in `en.yml`):**
```suggestion
      trusted: "%{username} now has a Trusted role!"
```

**Review by juliannatetreault — COMMENTED** [review]
I left a few suggestions and one clarifying question, otherwise this looks great so far, @yheuhtozr! 🌟

File: config/locales/controllers/admin/fr.yml [hunk 5]
```diff
@@ -1,18 +1,153 @@
...
+      privacy_policy:
+        title: Privacy Policy
+        description: A page that describes the privacy policy
+      terms_of_use:
+        title: Terms of Use
+        description: A page that describes the terms of use for the application
+    podcasts_controller:
+      new_owner_added: New owner added!
+      no_such_user: No such user
+      scheduled: 'Podcast''s episodes fetching was scheduled (%{title}, #%{id})'
```
Comment (juliannatetreault) [inline config/locales/controllers/admin/fr.yml:82]: One, final suggestion (same as above) that was missed!:
```suggestion
      scheduled: "Podcast's episodes fetching was scheduled (%{title}, #%{id})"
```

**Review by juliannatetreault — COMMENTED** [review]
I left one final suggestion that was missed in the last commit but once that is addressed, I think this should be good to go, @yheuhtozr! ✨

**Review by juliannatetreault — APPROVED** [review]
Thank you so much for tackling this, @yheuhtozr! 🌟

**Pull request merged**